package/build.gradle

```diff
@@ -0,0 +1,27 @@
+description = 'Sparkling Water Spark Package'
+
+apply plugin: 'com.github.johnrengelman.shadow'
+
+dependencies {
+    // hack to go around Spark Ivy resolver
+    compileOnly "no.priv.garshol.duke:duke:1.2"
+
+    compile project(':sparkling-water-core')
+    compile project(":sparkling-water-examples")
+    compile project(":sparkling-water-ml")
+    compile project(":sparkling-water-repl")
+}
+
+jar {
+    enabled = true
+}
+
+shadowJar {
+    archiveName = jar.archiveName
```

Comment:
@mmalohlava thanks for explanation. I thought that initial ida for doing this was to solve the problems similar to the Jetty one by including problematic shadowed dependencies.

Even though it's not a nice solution it would improve the user experience on Databrix and when using `--packages`